### DISCUSSION TITLE
Setting up CodeImprover

### DISCUSSION THREAD

**Discussion by 000xE** [CodeImprover/CodeImprover#14] [Q&A] [answered]
Hi, 

I installed CodeImprover on one of my repos although new commits doesn't seem to display any results from CodeImprover, I already have a workflow running automatically from GitHub actions for unit testing on each commit, does this affect it? If so, is there a workaround for this? Thanks.

**Comment by adjuc** [comment]
Hi 
I do not believe that the CodeImprover Metrics app check neither interferes with nor is disturbed by other GitHub apps or actions.
So the problem is probably caused by something else.
One of our tech experts will take a closer look and I will get back to you ASAP :-)
Regards Anders

**Comment by kasperbirkelund** [comment]
Hi,
I have found your installations and uninstallations in our log. At the moment your have one repository installed.

Please try to push a commit to the git repository starting with "1808" - any branch. By then you should see CodeImprover respond to the push and start analyzing your commit.

If you find troubles receiving the feedback from CodeImprover, I recommend that you write to support@codeimprover.com, then we can troubleshoot the issue more indepth :-)

**Comment by adjuc** [comment, marked as answer]
Case closed - we ended up correcting a minor error.
Thanks to outerme for the help :-)
